File: EqBench/CLEVER/ltfive2/EqWMain/newV.c
int lib(int x) {
	if (x < 0)
		return 0;
	else
		return x;
}
int client(int x){
	if (x < 0){
		return -lib((-x)*5)/5;
	}else{
		return lib((x+1)*5)/5 - 1;
	}
}

int main()
{
return 0;
}

File: EqBench/CLEVER/odd/Neq/newV.java
package benchmarks.CLEVER.odd.Neq;
public class newV {
    private int lib(int x) {
        int counter = 0;
        while (x % 2 == 0){
            x = x/2;
            counter++;
        }
        return counter+1;
    }
    int client(int x){
        if (lib(x)==0){
            return 1;
        }else{
            return 0;
        }
    }
}